migrations/README.md

```diff
@@ -80,9 +80,14 @@ Additionally, [supabase/postgres](https://github.com/supabase/postgres/blob/deve
 
 ```shell
```

Comment:
This instructions have changed and it is recommended to use 

`nix run .#dbmate-tool -- --version 15 --flake-url "."` on your local machine to reconcile migrations.

to create a new migration, you can run `nix develop` 

...and then `cd migrations` and `dbmate new '<some message>'` 

and this will make it possible to deprecate the docker file and docker set up here .

Reply:
I should have added this to this README in the past